data-store-api/config.py

```diff
@@ -41,6 +42,9 @@ class Config(BaseConfig):
     S3_STORAGE_BUCKET_NAME: str
     HANDLE_API_ENDPOINT: str
     REGISTRY_API_ENDPOINT: str
+    job_api_endpoint: str
```

Comment:
Does this "job_api_endpoint" need to be in all capital format?

Reply:
If anything, I'd prefer to change all of the others to lower case. That's the standard being used in most of the other APIs. 

Let's make this a separate issue to unify this across all the APIs - I know there's a mixture in other areas as well.